src/middleware/checkToken.ts

```diff
@@ -15,7 +15,7 @@ const checkToken = (req: Request, res: Response, next: NextFunction) => {
       console.log(err);
       return res.status(500).json({ error: 'Failed to authenticate token' });
     }
-
+    console.log('User ID is: ' , decoded);
```

Comment:
If I'm not wrong, I believe this console was for debugging . If it was , then you can remove it since the codes are working.